docs/source/installation.md

```diff
@@ -16,183 +16,182 @@ limitations under the License.
+<Tip>
+Add 🤗 [Datasets](https://huggingface.co/docs/datasets/) to your offline training workflow by setting the environment variable `HF_DATASETS_OFFLINE=1`.
+</Tip>
 
-On the instance with the normal network run your program which will download and cache models (and optionally datasets if you use 🤗 Datasets). For example:
+For example, you would typically run a program on a normal network firewalled to external world instances with the following command:
 
-```
+```bash
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
 
-and then with the same filesystem you can now run the same program on a firewalled instance:
-```
+Run this same program in an offline instance by:
+
+```bash
 HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1 \
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
-and it should succeed without any hanging waiting to timeout.
 
-#### Fetching models and tokenizers to use offline
+This should run the script without hanging or waiting to timeout.
 
-When running a script the first time like mentioned above, the downloaded files will be cached for future reuse. 
-However, it is also possible to download files and point to their local path instead.
+## Fetch models and tokenizers to use offline
 
-Downloading files can be done through the Web Interface by clicking on the "Download" button, but it can also be handled
-programmatically using the `huggingface_hub` library that is a dependency to `transformers`:
+Another option for using 🤗 Transformers offline is to download the files ahead of time, and then point to their local path when you need to use them offline. There are two ways to do this:
 
-- Using `snapshot_download` to download an entire repository
-- Using `hf_hub_download` to download a specific file
+* Download a file through the user interface on the [Model Hub](https://huggingface.co/models) by clicking on the ↓ icon.
 
-See the reference for these methods in the huggingface_hub
-[documentation](https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub).
+    ![download-icon](/imgs/download.png)
 
-## Do you want to run a Transformer model on a mobile device?
+* Programmatically download files with the [`huggingface_hub`](https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub) library:
 
-You should check out our [swift-coreml-transformers](https://github.com/huggingface/swift-coreml-transformers) repo.
+    1. Install the `huggingface_hub` library in your virtual environment:
 
-It contains a set of tools to convert PyTorch or TensorFlow 2.0 trained Transformer models (currently contains `GPT-2`,
-`DistilGPT-2`, `BERT`, and `DistilBERT`) to CoreML models that run on iOS devices.
+        ```bash
+        python -m pip install huggingface_hub
+        ```
```

Comment:
Nit: we only need to indent this under the 1. no?

Reply:
Indenting it further makes it a part of the preceding list item (looks a bit better this way I think) whereas indenting under `1.` would not:

<img width="478" alt="Screen Shot 2021-12-20 at 5 00 45 PM" src="https://user-images.githubusercontent.com/59462357/146843984-b15f17bd-afad-4f69-8c65-581e4bdb0657.png">